backend/eAkademik.API/Controllers/UserController.cs

```diff
@@ -71,4 +72,21 @@ public async Task<IActionResult> EditUser([FromBody] UserViewModel userViewModel
             return BadRequest("Couldn't update user");
         }
     }
-}
\ No newline at end of file
+    [HttpPost]
+    public async Task<ActionResult<UserViewModel>> CreateUser(string firstName, string lastName, string email, string password)
+    {
+        try
+        {
+            var user = await _userService.CreateUser(firstName, lastName, email, password);
```

Comment:
Kiedy dodasz nową klase viewModelu to metoda `CreateUser` również może przyjmować ją zamiast 4 argumentów. Ma to na celu uniknięcie sytuacji, że w przyszłości zmienią się wymagania i np. zamiast 4 parametrów bedzie 10 i metoda będzie mało czytelna i konieczne będzie zaktualizowanie wszystkich wywołań metody.